lua/formatter/filetypes/python.lua

```diff
@@ -39,6 +39,14 @@ function M.black()
   }
 end
 
+function M.ruff()
+  return {
+    exe = "ruff format",
+    args = { "-q", "-" },
```

Comment:
This is more in line with the other formatters, e.g. : https://github.com/mhartington/formatter.nvim/blob/master/lua/formatter/defaults/biome.lua
```suggestion
    exe = "ruff",
    args = { "format", "-q", "-" },
```